### DISCUSSION TITLE
It's not working for responsive

### DISCUSSION THREAD

**Discussion by IrynaVynnychenko** [tailwindlabs/tailwindcss#6743] [Ideas]
Hello, line-camp is not working for responsive:

line-clamp-5 xl:line-clamp-7

I've added in tailwind.config.js this for responsive, but still the same:

variants: {
    lineClamp: ['responsive', 'hover']
  },

**Comment by NovusOpus** [comment]
The reason why it's not working is because by default [line-clamp](https://github.com/tailwindlabs/tailwindcss-line-clamp/) only comes with classes for up to 6 lines, starting with `line-clamp-1` and ending in `line-clamp-6`. there's also `line-clamp-none` but that's for removing any line-clamping.

If you need 7 or more you need to extend the values you want in your` tailwind.config.js` like this:
```js
module.exports = {
  theme: {
    extend: {
      lineClamp: {
        7: '7',
        8: '8',
        9: '9',
        10: '10',
      }
    }
  },
  plugins: [
    require('@tailwindcss/line-clamp')
  ],
}
```
Also if you're using the new Tailwind CSS V3. Every variant is enabled out of the box, so you don't have to use this anymore:
```
variants: {
lineClamp: ['responsive', 'hover']
}
```
### And here's a `line-clamp` Working example with your values : [Tailwind Play](https://play.tailwindcss.com/pPXlBkxFpK?size=1300x720)
